### DISCUSSION TITLE
CircleCI: does anyone know what, if anything, it costs?

### DISCUSSION THREAD

**Discussion by ryjones** [open-quantum-safe/liboqs#1792] [General]
[WRT](https://github.com/open-quantum-safe/tsc/issues/24) [and](https://github.com/PQCA/TAC/issues/20), we need to figure out how much CircleCI OQS specifically, and PQCA in general, are spending/using. Minutes/month, dollars/month, anything would be helpful. @SWilson4 @dstebila ?

**Comment by baentsch** [comment]
You didn't tag me but allow me to respond regardless as the person mostly responsible for (setting up) all CI OQS has been using (and who introduced CircleCI a few years ago). We are paying nothing to CircleCI. We are most of the time staying well below their OSS free limits.

I obviously cannot comment on what PQCA is spending/using. My first priority over the past years has been to [be frugal](https://github.com/open-quantum-safe/tsc/issues/5) such as to allow OQS to scale up in contributions/PRs triggering CI use. I'm not aware whether other PQCA projects have the same mind set.

![grafik](https://github.com/open-quantum-safe/liboqs/assets/57787676/2d962c80-b2a1-43de-a145-baab36969129)

![grafik](https://github.com/open-quantum-safe/liboqs/assets/57787676/f5a47809-e037-4205-92b3-f785fb6906d1)

**Reply by ryjones** [reply]
excellent, thank you

**Comment by ryjones** [comment]
Thank you, @baentsch. this was what I needed.
